installer/debian/algorand/postinst

```diff
@@ -35,3 +35,11 @@ if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-decon
 		deb-systemd-invoke $_dh_action algorand.service >/dev/null || true
 	fi
 fi
+
+if [ "$1" = "configure" ]; then
+    if [ -f /var/lib/algorand/genesis.json-2.1.5-patch ]; then
+        # 2.1.5 bug fix - restore user-modified genesis.json
+        mv /var/lib/algorand/genesis.json{-2.1.5-patch,}
```

Comment:
I'd prefer to write this out longhand for readability, but that's just an opinion.

Reply:
Will revert.